apps/web/styles/app.css

```diff
@@ -76,3 +76,15 @@ body,
     display: none;
   }
 }
+
+@keyframes gradient {
```

Comment:
This is good enough for now, but in the future, it would be good if the keyframes definition could live next to the component. RNW should allows this via https://necolas.github.io/react-native-web/docs/styling/#non-standard-properties.